src/pydex_app/sra_routes.py

```diff
@@ -147,8 +146,8 @@ def post_order():
     current_app.logger.info(request.json)
     assert_valid(request.json, "/signedOrderSchema")
     order = SignedOrder.from_json(request.json, check_validity=True)
-    db.session.add(order)  # pylint: disable=no-member
-    db.session.commit()  # pylint: disable=no-member
+    Orderbook.add_order(order=order)
```

Comment:
I like this change and removal of the dependency, however please note my comments above mean for now the order-watcher client stuff should be removed so I can figure out how to put them in a separate process or a daemon which stays alive in between calls.

Reply:
nevermind, I realized you are only registering the order with the server here which is all that is needed.